### DISCUSSION TITLE
Allow to change the taskbar background colour to black

### DISCUSSION THREAD

**Discussion by jed84** [valinet/ExplorerPatcher#997] [Ideas]
Windows OS does not allow the user to set the background colour of the taskbar to black. 
It's possible to achieve by using 3rd party software, such as TaskbarX.

My request is to implement similar functionality in ExplroerPatcher so we can set the taskbar colour to #000000.

**Comment by valinet** [comment]
My take on this is to leave specialized utilities take care of this, like TranslucentTB. It has a host of other options, and after black taskbar, what follows will basically mean porting TranslucentTB to EP which I don’t see a need or reason for. If anyone wants to tackle that, maybe, but for now, I personally recommend using that.
